CHANGELOG.md

```diff
@@ -70,6 +70,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Fix compression support for h2c protocol ([#4944](https://github.com/opensearch-project/OpenSearch/pull/4944))
 - Support OpenSSL Provider with default Netty allocator ([#5460](https://github.com/opensearch-project/OpenSearch/pull/5460))
 - Added depth check in doc parser for deep nested document ([#5199](https://github.com/opensearch-project/OpenSearch/pull/5199))
+- Fix fuzziness validation ([#5805](https://github.com/opensearch-project/OpenSearch/pull/5805))
```

Comment:
@sejli could you please move this entry under `## [Unreleased 2.x]` section, the change will be backported to `2.x`, thank you

Reply:
Moved to the [unreleased 2.x section](https://github.com/opensearch-project/OpenSearch/pull/5805/commits/2980409d2e361ec149f04d64b4011c5c6ac4da49)